src/js/components/Select/stories/ClearValue.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Clear } from 'grommet-icons/icons/Clear';
+
+import { Box, Grommet, Select, Button } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Text } from '../../Text';
+
+const optionList = [
+  'One',
+  'Two',
+  'Three',
+  'Four',
+  'Five',
+  'Six',
+  'Seven',
+  'Eight',
+  'Nine',
+  'Ten',
+];
+
+const UnSelect = () => {
+  const [value, setValue] = useState('');
+
+  const onClickClearOptions = () => {
+    setValue('');
+  };
+
+  return (
+    <Grommet theme={grommet}>
+      <Box
+        pad="medium"
+        direction="row"
+        align="center"
+        justify="center"
+        gap="small"
+      >
+        <Box gap="small">
+          <Text>Default</Text>
+          <Select
+            options={optionList}
+            value={value}
+            onChange={({ option }) => setValue(option)}
+            placeholder="Select a value"
+            clear
+            multiple
+          />
+        </Box>
+        <Box gap="small">
+          <Text>Position Bottom</Text>
+          <Select
+            options={optionList}
+            value={value}
+            onChange={({ option }) => setValue(option)}
+            placeholder="Select a value"
+            multiple
```

Comment:
Interestingly, all the Selects on this story uses multiple but none are actually working, can you take a look at it? maybe it's because of the clear all binding.

Reply:
i just users wrong event handler. i ll just keep it as single.